collections/_posts/2021-07-09-how-to-pick-a-chia-pool.en.md

```diff
@@ -0,0 +1,70 @@
+
+Community engagement is also important for many people. This doesn’t necessarily mean engagement on a day-to-day level, but should you have problems, it’s good to have an easy way to get in touch with someone that can provide some kind of direction to support you in your time of need. Some providers are regularly available through services like Telegram or Discord or even through Facebook or Twitter.
+
+## Transparency and communication
+
+The transparency that a pool provides is another important factor. First and foremost is their payment structure for individual farmers' shares of the profits of pooled farming. How are they going to reward you for your part of the pool? It is very common for pools to operate on a percentage of each farmer's earnings. Running a pool isn't an easy task and takes considerable resources and technical knowledge to do it at a basic satisfactory level. For their work, it is only fair that a pool operator will charge you, the farmer, for providing the service that they do. Many farms have settled over time into a standard of setting fees in the 1-3% range of the rewards. Another thing to be aware of is how they structure their shares, when and how you are earning shares, and when those shares are no longer counted. 
+
+Also, consider what the process of leaving their pool is like–how long does it take to "free" your plots from being locked to the pool, and what happens to any shares that you have active?
+
+Another point to consider is who is behind the pool. Many of these people running a pool might not be willing to plaster their face and full name on their pool website. With a bit of searching, are you able to identify the entity behind the service? We say "entity" here because it may very well be something you need to consider regarding legislation and laws. If not already in place, new legislation may very well be coming in the near future. Knowing that you are using a service provided by a company, or person based in America, Europe or Asia, may be a factor for you. In the unfortunate case that should ever become necessary, it is also much easier to take an entity to court in a country where you both reside.
+
+## Fee payout structure
+
+We expect _PPLNS_ (Pay Per Last N Shares) to be the dominant pool rewards format. In this reward format, after a set period, any shares contributed to the pool in this time period are used to determine each farmer's payout. How long that period is can vary greatly as it is tied directly to when a pool wins a reward. _PPS_ (Pay Per Share) is another less common format where there is a predetermined time period, and the pool will keep a constant payout based on shares and the total pool performance. PPS is meant to eliminate large payout variance for the farmer, where the pool takes on the risks of any luck, bad or good. The two different formats have their pros and cons.
```

Comment:
Can we link to anything deeper on the two pool types?

Reply:
https://medium.com/luxor/mining-pool-payment-methods-pps-vs-pplns-ac699f44149f
ive been using this primarily for pooling schemes - is it acceptable as a link ? 
other resource i used was from nicehash:
https://www.nicehash.com/blog/post/how-mining-pools-distribute-rewards-pps-vs-fpps-vs-pplns

Reply:
I like that first link, let's use that one Roy.